demos/LowAllocationWebServer/LowAllocationServer.csproj

```diff
@@ -35,43 +35,44 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Buffers, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>packages\System.Buffers.0.1.0-d102115-1\lib\portable-net45+win8+wpa81+aspnetcore50\System.Buffers.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Core" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Text.Formatting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>packages\System.Text.Formatting.0.1.0.0-d100915-1\lib\portable-net45+win8+wpa81+aspnetcore50\System.Text.Formatting.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Text.Utf8, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>packages\System.Text.Utf8.0.1.0-d102115-1\lib\portable-net45+win8+wpa81+aspnetcore50\System.Text.Utf8.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
+    <ProjectReference Include="..\..\src\System.Text.Http\src\System.Text.Http.csproj">
+      <Name>System.Text.Http</Name>
+    </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="ByteSpanExtensions.cs" />
-    <Compile Include="HttpHeaderBuffer.cs" />
-    <Compile Include="HttpHeaders.cs" />
-    <Compile Include="HttpRequestParser.cs" />
-    <Compile Include="IFormatterHttpExtensions.cs" />
     <Compile Include="Log.cs" />
     <Compile Include="HttpServer.cs" />
     <Compile Include="main.cs" />
     <Compile Include="RoutingTable.cs" />
     <Compile Include="SampleServer.cs" />
     <Compile Include="SocketInterop.cs" />
-    <Compile Include="SpanExtensions.cs" />
     <Compile Include="TcpServer.cs" />
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Buffers">
+      <HintPath>..\..\..\packages\System.Buffers.0.1.0-d103015-1\lib\portable-net45+win8+wpa81+aspnetcore50\System.Buffers.dll</HintPath>
```

Comment:
We should add a project.json with these dependencies, so that we can build with the CLI. But I can do that after you merge your changes.